src/mme-app/backtrace.c

```diff
@@ -0,0 +1,206 @@
+#include <execinfo.h>
```

Comment:
adding copyright would be required. Or Skip this file I will take care of it.

Reply:
Will leave for you to update.